Node.cpp

```diff
@@ -1,9 +1,33 @@
 #include "Node.h"
+#include <limits>
 
-Node::Node(void) {
-    exists = true;
-}
+using namespace std;
+
+Node::Node(void):
+	pos(NodeCoordinate(0,0)),
+	exists(false)
+{}
+
+Node::Node(NodeCoordinate pos):
+	pos(pos),
+	exists(true)
+{}
 
 Node::operator bool() const {
     return exists;
 }
+
+void Node::resetPathData()
+{
+	gScore = numeric_limits<unsigned>::max();
+	fScore = numeric_limits<unsigned>::max();
+	previous = nullptr;
+	evaluated = false;
+	direction = NONE;
+}
+
+void Node::setScores(unsigned gScore, unsigned fScore)
```

Comment:
Is this method really necessary, since `gScore` and `fScore` are public members? (Or is there any additional processing we might want to add to this function in the future?)